### ISSUE TITLE
`--cloud_config` argument doesn't work with absolute paths

### ISSUE THREAD

**Issue by quinncomendant** [AntonioMeireles/coreos-xhyve#63]
I'm trying to `cloudctl run --cloud_config /usr/local/Cellar/corectl/0.5.4/share/cloud-init/docker-only-with-persistent-storage.txt` (the path to the cloud config file included in the brew package), but `corectl` appears to prepend the user's home to the supplied path. I get this error:

```
Error: open /Users/q/usr/local/Cellar/corectl/0.5.4/share/cloud-init/docker-only-with-persistent-storage.txt: no such file or directory
```

It works if I specify the path relative to home: `--cloud_config ../../usr/local/Cellar/corectl/0.5.4/share/cloud-init/docker-only-with-persistent-storage.txt`. =\

This problem doesn't exist with the `--volume` argument (that can be specified with a relative or absolute path).

**Comment by AntonioMeireles** [comment]
@quinncomendant this is fixed in [v0.7.x](https://github.com/AntonioMeireles/corectl/tree/v0.7.x) branch that will be in a mergeable state soon :-) . will close this then. 

thanks for reporting and have a great weekend!

**Comment by AntonioMeireles** [comment]
@quinncomendant  i believe that this is fixed in HEAD already. pleae reopen if it is not. (and thanks for your patience!)
